shared/metrics/src/main/java/io/pravega/shared/MetricsTags.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared;
+
+import com.google.common.base.Preconditions;
+
+public class MetricsTags {
+
+    // Metric Tag Names
+    public static final String TAG_CONTAINER = "container";
+    public static final String TAG_HOST = "host";
+    public static final String TAG_SCOPE = "scope";
+    public static final String TAG_STREAM = "stream";
+    public static final String TAG_SEGMENT = "segment";
+    public static final String TAG_TRANSACTION = "transaction";
+    public static final String TAG_EPOCH = "epoch";
+
+    private static final String TRANSACTION_DELIMITER = "#transaction.";
```

Comment:
Please take these out of this class and make use of methods in `StreamSegmentNameUtils`. All stream naming logic should reside in that class.

Reply:
I already tried. Metric code is on the lowest level of the dependency relationship defined, it doesn't have access to those classes.